src/test/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocPackageCheckTest.java

```diff
@@ -98,7 +98,7 @@ public void testBoth() throws Exception {
     public void testHtmlDisallowed() throws Exception {
         final Configuration checkConfig = createModuleConfig(JavadocPackageCheck.class);
         final String[] expected = {
-            "1: " + getCheckMessage(MSG_PACKAGE_INFO),
+            "8: " + getCheckMessage(MSG_PACKAGE_INFO),
         };
         verify(checkConfig,
```

Comment:
Please use another method, as defined in issue.

Reply:
@romani should I use ```verifyWithInlineConfigParser()``` instead of ```verify()``` method. Because right now I am getting error when I run test in intellij : 


```Violations for C:\Users\shubh\OneDrive\Desktop\checkstyle project\checkstyle\src\test\resources\com\puppycrawl\tools\checkstyle\checks\javadoc\javadocpackage\InputJavadocPackageBadCls.java differ.
value of: iterable.onlyElement()
expected: 8: Missing package-info.java file.
but was : 1: Missing package-info.java file.
Expected :8: Missing package-info.java file.
Actual   :1: Missing package-info.java file.
```

The package has to be on line number 1 mandatorily as it has been hardcoded in the JavadocPackageCheck.java

https://github.com/checkstyle/checkstyle/blob/469a4a8f306348428026c6f94d10d9d77388b8f0/src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocPackageCheck.java#L132-L139

What am I supposed to do after this.

Reply:
>should I use verifyWithInlineConfigParser() instead of verify() method.

Yes 

Check does not care about package. It just report violay on first line.

So you can use comment after config line `//violation 6 lines abone`

Reply:
@romani 

1. I have placed the package at line number 1.

2. the config is placed right below the package line.

3. The violation message is below the config. The package line is 7 lines above the violation message.


**Now the problem is ```verifyWithInlineConfigParser()``` is giving me errors as follows:**

```
com.puppycrawl.tools.checkstyle.api.CheckstyleException: Config comment not specified properly in C:\Users\shubh\OneDrive\Desktop\checkstyle project\checkstyle\src\test\resources\com\puppycrawl\tools\checkstyle\checks\javadoc\javadocpackage\InputJavadocPackageBadCls.java

```

Reply:
@romani not placing the config on the top might produce the error:

`
Config not specified on top.Please see other inputs for examples of what is required.
`

Reply:
Test failing because line number might change, we need to update test expected values